Source/AFError.swift

```diff
@@ -466,6 +552,14 @@ extension AFError: LocalizedError {
             return "Server trust evaluation failed."
         case let .urlRequestValidationFailed(reason):
             return "URLRequest validation failed due to reason: \(reason.localizedDescription)"
+        case let .createUploadableFailed(error):
+            return "Creating uploadable failed with error: \(error.localizedDescription)"
+        case let .createURLRequestFailed(error):
+            return "Creating URLRequest failed with error: \(error.localizedDescription)"
+        case let .downloadedFileMoveFailed(error, destination):
+            return "Moving download to \(destination) failed with error: \(error.localizedDescription)"
```

Comment:
Perhaps "Moving downloaded file to"?

Reply:
I decided to make this multiline after adding the `source` value. Let me know what you think.